.gitignore

```diff
@@ -5,3 +5,4 @@
 /.idea/
 *~
 /**/target
+/home
```

Comment:
Please include a newline at the end of the file.

Reply:
I'm reasonably certain this is fixed now, but I squashed the commits for sanity's sake and the web interface makes it impossible to tell that anything was actually changed. But it looks right on a fresh local clone, so...